### ISSUE TITLE
symbols packages not published

### ISSUE THREAD

**Issue by dmelinosky** [LarsSkovslund/NuGet.Package.Builder#9]
If you generate symbols packages with your normal package, they are not published with the normal package files.

**Comment by Meandron** [comment]
I support that request - would be great to be able to automatically publish symbols on build.

**Comment by LarsSkovslund** [comment]
I have now added support for publishing symbols along side the package in release 1.0.10.

SymbolSource can be overridden though msbuild property and json file.

Please give this a try and let me know if you find any issues.

Cheers
 Lars

**Comment by Meandron** [comment]
Hi Lars! I tried it out - works perfectly for me! Thanks a lot!

Regards
Meandron

**Comment by dmelinosky** [comment]
I'm having an issue.   The symbol source I'm using is generating the following error:

The controller for path '/api/v2/package/' could not be found or it does not implement IController.

I used the Nuget Package SymbolSource.Server.Basic to create my own internal site to host the symbols.   I set the SymbolSource configuration option to "http://symbolsource.mydomain.com" and "http://symbolsource.mydomain.com/nuget" and get the same result.  The command it says I should use to push is 

nuget push *.symbols.nupkg 123 -Source http://symbolsource.mydomain.com/NuGet

I'm still trying to get a nugget.exe on my system to try this via the command line so I can report back if I'm able to push myself.

Will let you know.
